src/test/unit/controls/Graph/GraphGenerate-spec.js

```diff
@@ -507,10 +507,39 @@ describe("Graph - Generate", () => {
                 axisDefault.x.label
             );
         });
-        it("Creates label y axis when text is present", () => {
+        it("Creates label y axis when y2-axis is false and when text is present", () => {
```

Comment:
nit: the wording little confusing. What is text present? Is that y-axis label present?

nit: It can be updated like below
```js
describe("when y2-axis is false and y-axis label is present", () => {
     it("should create y-axis label and reduce the gap between y-axis and y-axis label", () => {
    });
});
``

Reply:
Well that sounds good. But in order to maintain the consistency with previous test cases i just wrote it this way.